xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/client/accounts/AccountCurrenciesResultJsonTests.java

```diff
@@ -0,0 +1,45 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.primitives.UnsignedInteger;
+import com.google.common.primitives.UnsignedLong;
+import org.json.JSONException;
+import org.junit.Test;
+import org.xrpl.xrpl4j.model.AbstractJsonTest;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+import org.xrpl.xrpl4j.model.transactions.Marker;
+import org.xrpl.xrpl4j.model.transactions.XrpCurrencyAmount;
+
+public class AccountCurrenciesResultJsonTests extends AbstractJsonTest {
```

Comment:
Can we add a test here to make sure the JSON works when requesting with `"ledgerIndex": "current"`? We've run into problems with other result objects with differences between what rippled sends back for the current ledger.

I would just use PostMan or Curl to send a request to rippled and copy/paste the response into a test.

Reply:
tested it with `current` using xrpl-py - works fine for a response. not quite sure if that's what you were asking?

Reply:
yeah that's probably good enough